x-pack/plugins/enterprise_search/public/applications/app_search/components/create_engine/create_engine_logic.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { kea, MakeLogicType } from 'kea';
+import { generatePath } from 'react-router-dom';
+
+import { CREATE_ENGINE_SUCCESS_MESSAGE } from './constants';
+import { ENGINE_PATH } from '../../routes';
+import { formatApiName } from '../../utils/format_api_name';
+import { flashAPIErrors, setQueuedSuccessMessage } from '../../../shared/flash_messages';
+import { HttpLogic } from '../../../shared/http';
+import { KibanaLogic } from '../../../shared/kibana';
+
+export const DEFAULT_LANGUAGE = 'Universal';
```

Comment:
What do you think about storing this is `constants.ts` (above or below SUPPORTED_LANGUAGES) as opposed to a one-off kinda just hanging out at the top of the file?